Testing/functional/e2e/debug.log

```diff
@@ -0,0 +1,9 @@
+[1105/114616.578:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
This file should be added to the git ignore.

Reply:
@sslaws good catch, I have added it to the git ignore